lib/modules/home/components/slide_item.dart

```diff
@@ -2,21 +2,49 @@ part of '../home_module.dart';
 
 class SlideItem extends StatelessWidget {
   final SurveyInfo survey;
+  final bool _isEmptied;
+
   const SlideItem({
     Key? key,
     required this.survey,
-  }) : super(key: key);
+  })  : _isEmptied = false,
+        super(key: key);
+
+  const SlideItem._({
+    Key? key,
+    required this.survey,
+    bool isEmptied = false,
+  })  : _isEmptied = isEmptied,
+        super(key: key);
+
+  factory SlideItem.empty({
+    Key? key,
+  }) {
+    final survey = SurveyInfo();
+    survey.title = "";
+    survey.description = "";
+    survey.coverImageUrl = "";
+    return SlideItem._(
+      key: key,
+      survey: survey,
+      isEmptied: true,
+    );
+  }
 
   @override
   Widget build(BuildContext context) {
+    final state = context.findAncestorStateOfType<_HomeViewImplState>()!;
     return Stack(
       fit: StackFit.expand,
       children: [
         Bone.hidden(
-          child: Image(
-            image: NetworkImage(survey.coverImageUrl!),
-            fit: BoxFit.cover,
-          ),
+          child: _isEmptied
```

Comment:
can check `survey.title == null`, or ideally we can create an extension function 
`survey.isEmpty() => survey.title == null && survey.description == null && survey.imageUrl == null`
then use `survey.isEmpty()` here

Reply:
I think you misunderstand my purpose here. I create a `_isEmptied` to flag this SlideItem as an empty slide, so I can use it to adapt skeleton animation on it 😁

Reply:
ah, as you defined `get _isEmptied => survey == null;`, it makes sense now 🙏